packages/core/src/interfaces/components.ts

```diff
@@ -141,3 +145,21 @@ export interface ZoomBarOptions {
 	 */
 	data?: Object[];
 }
+
+/**
+ * customize the ToolBar component
+ */
+export interface ToolBarOptions {
+	/**
+	 * a variable to handle zoom in ratio
+	 */
+	zoomRatio?: number;
+	/**
+	 * is the tool-bar visible or not
+	 */
+	showToolBar?: boolean;
+	/**
+	 * toolbar overflow menu options
+	 */
+	overflowMenuItems?: string[];
```

Comment:
this should be a typed object with an interface.

e.g. `overflowMenuItems?: ToolbarOverflowMenuItems;`

```ts
export interface ToolbarOverflowMenuItems {
	resetZoom?: boolean;
	print?: boolean; // we don't need to add this one yet
}
```

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/717/commits/50a9137867775ff491405b4b22971d9968682fa8.

We like to allow user to set their own string (mainly for i18n) for each menu item, so there will be a `text` prop for each menu item.

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/commit/50a9137867775ff491405b4b22971d9968682fa8.